app/helpers/format-time-duration.ts

```diff
@@ -0,0 +1,12 @@
+import { helper } from '@ember/component/helper';
+import { formatDuration, intervalToDuration } from 'date-fns';
+
+export default helper(function formatTimeDuration([timeDurationInSeconds]: [number] /*, named*/) {
+  const duration = intervalToDuration({ start: 0, end: timeDurationInSeconds * 1000 });
+
+  if (timeDurationInSeconds > 60) {
+    duration.seconds = 0;
```

Comment:
This seems kind of specific to the current use-case... If we had another place where we wanted to show something like "time since last attempt", it's possible that we want a more accurate timestamp like "1m 34s"

Can we try to make the interface more generic? I'd look at other similar functions you can find in OSS repos to figure out what a good interface could be.

Reply:
If naming is hard here, I'd instead recommend doing this logic on `@screencast`. You could add `@screencast.roundedDurationInSecondsForDisplay` or something like that - and have it return the actual value if < 1m, and the rounded value if > 1m.